src/dstr-binding/default/async-gen-func-named-expr-dflt.template

```diff
@@ -19,7 +19,7 @@ flags: [async]
 
 var callCount = 0;
 var f;
-f = async function* g(/*{ elems }*/ = /*{ vals }*/) {
+f = async function* h(/*{ elems }*/ = /*{ vals }*/) {
```

Comment:
I'm unclear on what the point of this is. Is `g` used by `{ body }` or `{ vals }` or something?